Orange/evaluation/testing.py

```diff
@@ -22,20 +24,27 @@ def _identity(x):
 
 def _mp_worker(fold_i, train_data, test_data, learner_i, learner,
                store_models):
-    predicted, probs, model, failed = None, None, None, False
+    predicted, probs, model, failed, train_time, test_time = None, None, None, False, None, None
     try:
         if not train_data or not test_data:
             raise RuntimeError('Test fold is empty')
+        # training
+        t0 = time()
```

Comment:
This is a bit crude. It can work for now, but it would be much better to measure the actual execution time of the thread, if possible.

Reply:
In python 3.7 there is [thread_time](https://docs.python.org/3/library/time.html#time.thread_time), in python 3.6 we have [process_time](https://docs.python.org/3/library/time.html#time.process_time). Do you think the latter will do?